javascript/sort_index_by_record_size.js

```diff
@@ -0,0 +1,82 @@
+/*
+Sort Index By Record Size
+Sometimes we want to easily find the largest record in an index (in file size) so we can investigate situations where some small number of records are over the fileSizeLimit. This script is designed to fetch the entire index and then sort it by the total string size, and then export it to a file for analysis.
+*/
+
+// Install the API client: https://www.algolia.com/doc/api-client/getting-started/install/javascript/?client=javascript
+const algoliasearch = require("algoliasearch");
+const dotenv = require("dotenv");
+
+dotenv.config();
+
+// Get your Algolia Application ID and (admin) API key from the dashboard: https://www.algolia.com/account/api-keys
+// and choose a name for your index. Add these environment variables to a `.env` file:
+const ALGOLIA_APP_ID = process.env.ALGOLIA_APP_ID;
+const ALGOLIA_API_KEY = process.env.ALGOLIA_API_KEY;
```

Comment:
Can't be run using search api key, I think browseObjects requires an admin API key